source/app/api.py

```diff
@@ -154,6 +154,11 @@ def add_disk(disk):
                           'asd_id': asd_id,
                           'log_level': 'info',
                           'port': port}
+            
+            if config.data['main'].has_key('__sync_dont_use'):
+                asd_config['__sync_dont_use'] = config.data['main']['__sync_dont_use']
```

Comment:
it's cleaner if you:
- avoid using the literal '__sync_dont_use' three times.
- use a `.get(...)` and check for None iso `.has_key(....)` 

iirc python has a conditional assignment: `x = y if z`